.gitignore

```diff
@@ -8,4 +8,5 @@ __pycache__
 .ipynb_checkpoints
 
 build
-dist
\ No newline at end of file
+dist
+docs/reference/.#index.rst
```

Comment:
It looks like the `.#` file is from your text editor. Could you please remove this from the PR.